tests/strategies/external/test_pv.py

```diff
@@ -41,6 +43,22 @@ def settlement_market_fixture():
                             time_slot=datetime(2021, 12, 7, 12, 00))
 
 
+@pytest.fixture(name="future_markets")
```

Comment:
Is it possible to reuse the fixture among different strategies and do not repeat it everywhere?

Reply:
I was also thinking about this but remembered something from @fievelk who said that each test module should be self-sufficient. Is that correct?

Reply:
It really depends on the case. Re-using code in tests can be fine. A fixture can be shared, if we know that many of our tests require that object (that's also one of the reasons why we use factories in gsy-web). Also, fixtures are made exactly with that purpose in mind: being shared.

It's often better to hardcode values inside of tests, rather than create a method that generates them in a complex way. That's because otherwise we would need to write tests for that method too!

What should be avoided is adding logic inside tests. For example, tests should not contain `if` statements: each test should only check one "flow", and it should do it in an obvious way (without behaving differently depending on conditionals).

Not sure if this answers the current situation :)

Reply:
Great! I moved the fixtures in their own module here: [d720f58](https://github.com/gridsingularity/gsy-e/pull/1537/commits/d720f58d73e5b921a9c4454cbe913b2f4fd8def9)